Editor/Scripts/Analytics/MeshsyncAnalytics.cs

```diff
@@ -72,12 +129,21 @@ private struct SyncEventData {
                     MESHSYNC_SYNC_VERSION));
         }
 
+
+        /// <inheritdoc />
         public void UserInstalledPlugin(string dccTool) {
             var data = new DCCInstallEventData { meshSyncDccName = dccTool.ToLower() };
             logIfWarning(EditorAnalytics.SendEventWithLimit(DCCCONFIGURE_INSTALLEDEVENTNAME, data, DCCCONFIGURE_VERSION));
         }
 
+        /// <inheritdoc />
         public void UserSyncedData(MeshSyncAnalyticsData data) {
+            if (data.entityType == EntityType.Unknown &&
+                data.assetType == AssetType.Unknown) {
+                Debug.LogWarning("Invalid event data sent");
```

Comment:
Our users won't know what to do when they see this log, so I'd say we should remove this log